plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAChecker.java

```diff
@@ -71,6 +73,29 @@ public Boolean checkingHeartBeat() {
             }
         }
 
+        for (RbdStoragePool pool : rbdStoragePools) {
+            Script cmd = new Script(s_heartBeatPathRbd, heartBeatCheckerTimeout, s_logger);
+            cmd.add("-i", pool._poolSourceHost);
+            cmd.add("-p", pool._poolMountSourcePath);
+            cmd.add("-n", pool._poolAuthUserName);
+            cmd.add("-s", pool._poolAuthSecret);
+            cmd.add("-h", hostIp);
```

Comment:
I know we use this with NFS, but isn't it better to use the UUID of the host here?

Each host has a unique UUID and that seems safer to use then the Private IPv4 address of the host.